third_party/docfx/templates/devsite/partials/uref/namespace.tmpl.partial

```diff
@@ -18,7 +18,7 @@
   {{/inTypes}}
   {{^isEmbedded}}
     {{#children}}
-      <h2><xref uid="{{uid}}" altProperty="fullName" displayProperty="name"/></h2>
+      <h3><xref uid="{{uid}}" altProperty="fullName" displayProperty="name"/></h3>

```

Comment:
Should line 44 (the other `h2`) also be bumped to `h3`?

Reply:
Has no effect for Python, if we adjust it that'll be for Go only, I believe only Go has `{{inTypes}}` enabled.